README.md

```diff
@@ -201,19 +180,18 @@ All features are documented in the [User Documentation](docs/user/README.md).
 ### Containers
 
 The operator is comprised of the following container images working in concert:
-* The operator, built from sources in the [operator](operator/) directory.
+* The operator, built from sources using the kubebuilder v3 structure, from [controllers](controllers/) directory.
 * The config builder init container, built from sources in [datastax/cass-config-builder](https://github.com/datastax/cass-config-builder).
 * Cassandra, built from
   [datastax/management-api-for-apache-cassandra](https://github.com/datastax/management-api-for-apache-cassandra),
-  with Cassandra 3.11.7 support, and experimental support for Cassandra
-  4.0-beta1.
+  with Cassandra 3.11.10 support, and limited support for Cassandra 4.0.
 * ... or DSE, built from [datastax/docker-images](https://github.com/datastax/docker-images).
 
 ### Overriding properties of cass-operator created Containers
 
 If the CassandraDatacenter specifies a podTemplateSpec field, then containers with specific names can be used to override default settings in containers that will be created by cass-operator.
 
-Currently cass-operator will create an InitContainer with the name of "server-config-init". Normal Containers that will be created have the names "cassandra", "server-system-logger", and optionally "reaper". 
+Currently cass-operator will create an InitContainer with the name of "server-config-init". Normal Containers that will be created have the names "cassandra" and "server-system-logger".
```

Comment:
A couple minor nits.

I would change `InitContainer` to `init container` since there is no InitContainer type/kind. 

I would change `Normal Containers` to `Normal containers`. While `Container` is a Go type, I would go with lower case unless we are specifically referring to the Go type.

Reply:
Yeah, I haven't written this, I just removed the reaper part. I can modify the words.